dev-util/bats/bats-1.3.0.ebuild

```diff
@@ -13,12 +13,21 @@ SRC_URI="https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 ppc64 ~riscv x86"
+IUSE="test"
```

Comment:
Declaring the 'test' use flag is not necessary if there are no test dependencies. See also https://devmanual.gentoo.org/ebuild-writing/functions/src_test/index.html#supporting-test-suites-in-packages

Reply:
unless i patch unconditionally this is needed because of the `if use test`